tensorflow/core/framework/shape_inference.cc

```diff
@@ -181,7 +182,8 @@ void InferenceContext::PreInputInit(
 }
 
 Status InferenceContext::ExpandOutputs(int new_output_size) {
-  if (new_output_size < outputs_.size()) {
+  int outputs_size_ = outputs_.size();
```

Comment:
Should be `const`

Reply:
Also, should not end with `_` as names ending in `_` are reserved for private class members in the coding style

Reply:
Not `const`, still ending in `_`